core/src/main/scala/com/banno/kafka/publish/PublishStreamOps.scala

```diff
@@ -0,0 +1,18 @@
+package com.banno.kafka.publish
+
+import cats.Applicative
+import cats.effect.Concurrent
+import fs2.Stream
+
+final class PublishStreamOps[F[_], A](private val stream: Stream[F, F[A]]) extends AnyVal {
+
+  def batched(implicit A: Applicative[F]): Stream[F, A] =
+    stream.evalMapChunk(fa => fa)
+
+  def batchedPar(maxConcurrent: Option[Int])(implicit A: Applicative[F], C: Concurrent[F]): Stream[F, A] =
```

Comment:
I think we can remove the Applicative constraint since it's entailed by Concurrent?

Reply:
ah yes silly me